client/src/ResouresPortalContext.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export const ResourcesPortalContext = React.createContext([{}, () => {}])
```

Comment:
This could should probably just be an empty object for now. It is supposed to resemble what the structure of the object will look like but the way the provider is designed it wont really affect anything.